grasp_planning_service:
  ros__parameters:
    gripper_offset: 0.02
    finger_width: 0.01
    max_results: 3
    camera_frame: camera_213322070245_color_optical_frame
masked_point_cloud:
  ros__parameters:    
    rgb_topic: /camera_213322070245/color/image_raw
    depth_topic: /camera_213322070245/aligned_depth_to_color/image_raw
    camera_info_topic: /camera_213322070245/color/camera_info
    camera_frame: camera_213322070245_color_optical_frame
    cutoff_volume: [0.4, 1.3, -0.6, 0.6, -0.02, 0.35]

    
    